backend/src/main/java/codezap/template/dto/response/FindTemplateResponse.java

```diff
@@ -45,7 +45,9 @@ public record FindTemplateResponse(
         @Schema(description = "템플릿 수정 시간", example = "2024-11-11 12:00:00", type = "string")
         LocalDateTime modifiedAt
 ) {
-    public static FindTemplateResponse of(Template template, List<SourceCode> sourceCodes, List<Tag> tags, Long likeCount, Boolean isLiked) {
+    public static FindTemplateResponse of(Template template, List<SourceCode> sourceCodes, List<Tag> tags,
+            Boolean isLiked
+    ) {
```

Comment:
개행을 return 값과 일관성 있게 하면 어떨까 제안해요~

```suggestion
    public static FindTemplateResponse of(
            Template template,
            List<SourceCode> sourceCodes,
            List<Tag> tags,
            Boolean isLiked
    ) {
```